test/conftest.py

```diff
@@ -12,37 +12,67 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from typing import Dict
+from typing import Any, Dict, List, TypeAlias
 
 import pytest
 
+_ConfSingleValue: TypeAlias = bool | int | float
+ConfigValue: TypeAlias = _ConfSingleValue | List[_ConfSingleValue] | Dict[str, _ConfSingleValue]
+
 
 @pytest.fixture
-def fast_config() -> Dict[str, Dict[str, bool | int | float]]:
+def fast_config_modifications() -> Dict[str, ConfigValue]:
```

Comment:
A stupid question: why `fast_config_modifications` instead of  `fast_config` 🙄

Reply:
switched back